assets/js/SortFromTable.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import {store, setSelectedWfModuleAction} from "./workflow-reducer";
+import {getPageID} from "./utils";
+import WorkbenchAPI from './WorkbenchAPI'
+import {findModuleWithIdAndIdName, findParamValByIdName, getWfModuleIndexfromId} from './EditCells'
```

Comment:
If these are generally useful they need to be moved out of EditCells and into something else. utils.js? findThings.js?